cl/assets/static-global/css/override.css

```diff
@@ -1114,3 +1121,18 @@ emphasis {
   text-indent: 10px;
 }
 /* END Tax specific css tweak*/
+
+/*Improve contrast on alert boxes*/
+.alert-dismissible .navbar-text.lead, .alert-danger, .alert-warning,
+.alert-danger .help-block {
+  color: #333333;
+  font-family: Helvetica;
```

Comment:
I added the font setting to force to use Helvetica since the default font in bootstrap is "Helvetica Neue" which is a bit thinner than Helvetica, but I didn’t know that Helvetica is only available on OSX, so I think is not worth changing it and better continue with the default bootstrap fonts, removed the font setting.

Reply:
Yeah, tinkering with font families is hard. I just saw this the other day: https://systemfontstack.com/